framework/Models.py

```diff
@@ -48,7 +48,7 @@
 from utils.cached_ndarray import c1darray
 from Csv_loader import CsvLoader
 import Files
-import PostProcessors
+import Processors
```

Comment:
Probably best to leave it as PostProcessors for now. I will move the folders. This change can be made quite easily later on when a name is agreed upon.